CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Composer script for PHP linting
 
 ### Changed
+- Moved PHP code sniffing to GitHub actions (#13)
```

Comment:
In general: Moved -> Move

Reply:
I tend to disagree, since the headline is also past tense. But then we should also fix all existing entries.

Reply:
I usually copy the Git commit message headlines into the changelog, resulting in this form.
https://chris.beams.io/posts/git-commit/

(But I'm not very passionate about the tenses in the changelog.)

Reply:
Sounds logical, let's stick to present tense then.